build.sh

```diff
@@ -9,6 +9,11 @@ then
 	source /config.env
 fi
 
+if [[ -f /secure/config.env ]]
```

Comment:
You know, I tried this, but I think I tried to create a `VOLUME` in the `Dockerfile` which can only accept a directory, but this seems to work, so 🎉

With this in mind, we can probably drop the whole "you need to build for yourself" part of the `README.md` when using your own config. Do you want to make those updates?